src/clib/lib/job_queue/job_list.cpp

```diff
@@ -1,66 +1,34 @@
 #include <pthread.h>
-#include <stdlib.h>
-#include <string.h>
-#include <unistd.h>
-
-#include <ert/util/util.hpp>
+#include <vector>
 
 #include <ert/job_queue/job_list.hpp>
 #include <ert/job_queue/job_node.hpp>
 
 struct job_list_struct {
-    int active_size;
-    int alloc_size;
-    job_queue_node_type **jobs;
-    pthread_rwlock_t lock;
+    std::vector<job_queue_node_type *> vec_jobs;
+    pthread_rwlock_t lock = PTHREAD_RWLOCK_INITIALIZER;
```

Comment:
I think this would have to be done differently. As far as documentation says, `PTHREAD_RWLOCK_INITIALIZER` can only be used on static and global objects.

Reply:
Initialising using brace-style should be equivalent to _init_.